src/scripts/build.sh

```diff
@@ -1,25 +1,27 @@
 #!/bin/bash
 set -o noglob
+ORB_STR_REGION="$(circleci env subst "${ORB_STR_REGION}")"
+ORB_STR_TEMPLATE="$(circleci env subst "${ORB_STR_TEMPLATE}")"
+ORB_EVAL_BUILD_DIR="$(eval echo "${ORB_EVAL_BUILD_DIR}")"
+ORB_EVAL_BASE_DIR="$(eval echo "${ORB_EVAL_BASE_DIR}")"
 
-set -- "$@" --profile "$(eval "echo $SAM_PARAM_PROFILE")"
+set -- "$@" --template-file "${ORB_STR_TEMPLATE}"
 
-set -- "$@" --template-file "$(eval "echo $SAM_PARAM_TEMPLATE")"
+set -- "$@" --region "${ORB_STR_REGION}"
 
-set -- "$@" --region "${!SAM_PARAM_AWS_REGION}"
-
-if [ -n "$SAM_PARAM_BUILD_DIR" ]; then
-  set -- "$@" --build-dir "$(eval "echo $SAM_PARAM_BUILD_DIR")"
+if [ -n "$ORB_EVAL_BUILD_DIR" ]; then
+  set -- "$@" --build-dir "${ORB_EVAL_BUILD_DIR}"
 fi
-if [ -n "$SAM_PARAM_BASE_DIR" ]; then
-  set -- "$@" --base-dir "$(eval "echo $SAM_PARAM_BASE_DIR")"
+if [ -n "$ORB_EVAL_BASE_DIR" ]; then
+  set -- "$@" --base-dir "${ORB_EVAL_BASE_DIR}"
 fi
-if [ "$SAM_PARAM_CONTAINER" = 1 ]; then
+if [ "$ORB_BOOL_USE_CONTAINER" = 1 ]; then
```

Comment:
Consider using `-eq` if the left-side value is a number. See https://stackoverflow.com/questions/18668556/how-can-i-compare-numbers-in-bash

Reply:
Good tip but the left side in this case, `ORB_BOOL_USE_CONTAINER` is a boolean value.